CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(metadata_dir ${CMAKE_SOURCE_DIR}/source/codegen/metadata)
 set(service_output_dir ${CMAKE_SOURCE_DIR}/generated)
 set(codegen_dir ${CMAKE_SOURCE_DIR}/source/codegen)
 set(custom_dir ${CMAKE_SOURCE_DIR}/source/custom)
-set(nidrivers "nifake" "niscope" "niswitch" "nisync")
+set(nidrivers "nidmm" "nifake" "niscope" "niswitch" "nisync")
```

Comment:
Preserve this order throughout the file for consistency. That is, keep the list of drivers in alphabetical order in the other places you've added `nidmm` as well.

Reply:
arranged it in alphabetical order